apps/metabase/0.1.0/metabase.application.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: metabase-deployment
+spec:
+  components:
+    - name: metabase
+      dependsOn:
+        - metabase-db
+      type: webservice
+      properties:
+        image: metabase/metabase
+        cpu: "250m"
+        memory: "4Gi"
+        ports:
+          - port: 3000
+            expose: true
+        env:
+          - name: MB_DB_TYPE
+            value: postgres
+          - name: MB_DB_DBNAME
+            value: metabasedb
+          - name: MB_DB_PORT
+            value: "5432"
+          - name: MB_DB_USER
+            value: metabase
+          - name: MB_DB_PASS
+            value: metabase
+          - name: MB_DB_HOST
+            value: metabase-db
+      traits:
+        - type: napptive-ingress
+          properties:
+            name: metabase-ingress
+            port: 3000
+            path: /
+    - name: metabase-db
+      type: webservice
+      traits:
+        - type: storage
+          properties:
+            pvc:
+              - name: penpot-postgres-pvc
+                mountPath: /var/lib/postgresql/data
+                subPath: postgres
+                resources:
+                  requests:
+                    storage: 1Gi
+      properties:
+        image: "postgres:15"
+        cpu: "50m"
+        memory: "512M"
+        ports:
+          - port: 5432
+            expose: true
+        env:
+          - name: POSTGRES_INITDB_ARGS
+            value: --data-checksums
+          - name: POSTGRES_DB
+            value: metabasedb
+          - name: POSTGRES_USER
+            value: metabase
+          - name: POSTGRES_PASSWORD
+            value: metabase
+    - name: metabase-test-db
```

Comment:
Why do we have two databases?

Reply:
We have two databases, one for metabase-related information, and the other for actually storing target analytics data as we don't want both to clash.

Reply:
![Screenshot 2023-11-21 at 10 31 54](https://github.com/napptive/catalog-apps/assets/5525160/85154e99-3e85-41d2-997b-c8af9c7d6d77)
So, what database should we configure here? Could you add it to the README file?

Reply:
The second one is marked as "metabase-test-db". Sure I'll add it to the Readme.

Reply:
Added